Assets/Scripts/Character/Enemies/Enemy.cs

```diff
@@ -14,7 +14,7 @@ public abstract class Enemy : MonoBehaviour, IPoolable, IDamageDealer
         [SerializeField]
         private int maxHealth = 3;
         [SerializeField]
-        private Enums.EnemyTypes type = Enums.EnemyTypes.Basic;
```

Comment:
Don't change this its supposed to be set in the editor not in the code.  If you are wondering why its because being an editor variable allows us to share scripts between different enemies.